tests/torch/requirements.txt

```diff
@@ -14,4 +14,7 @@ transformers==4.23.1
 # Required for movement_sparsity tests
 datasets==2.6.2
 evaluate==0.3.0
-timm
\ No newline at end of file
+timm
+
+# Use case with quantization of nn.MultiheadAttention
+open_clip_torch==2.18.0
```

Comment:
hope it will not increase the test time which already takes time as far as I heard. Potentially, you could create a model with one primitive to the text of use something from torchvision which should be more aligned with torch versions because now your test depend on the stability of the OpenCLIP library.

Reply:
It takes the specific version of the library, shouldn't depend much. 
test takes 11 seconds

Reply:
There are many conditions checked to make a decision about implementation of MHA
![image](https://user-images.githubusercontent.com/4014476/236442195-42afd426-915c-4d33-9690-9c2e701828b6.png)

Reply:
Removed dependency from open_clip